### DISCUSSION TITLE
Jira and Bitbucket Feature Set

### DISCUSSION THREAD

**Discussion by Cjewett** [renovatebot/renovate#23741] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.35.0 which is latest from the Helm chart

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We're looking into Renovate because it seems like a more natural fit with our ecosystem when compared with Dependabot. The Helm chart makes it a lot easier for us to introduce it to our system. One piece I'm confused on is the Jira support though. 

I see that there are multiple issues open to support Bitbucket and Jira from this [doc](https://docs.renovatebot.com/faq/#major-platform-features-not-supported-by-renovate):
https://github.com/renovatebot/renovate/issues/9592
https://github.com/renovatebot/renovate/issues/20568
https://github.com/renovatebot/renovate/issues/3796

I'm curious what features _do_ work because it sounds like there is no Dependency Dashboard or Jira support. If you run Renovate against Bitbucket Cloud, Server, or Data Center is the only feature that is supported at the moment that it creates a pull-request for that repository?

To add some context on what we're looking for we're a Bitbucket Data Center and Jira Data Center shop. When we run Renovate and it detects package upgrades our hope is that it creates a Bitbucket PR and in addition to that a Jira issue in a Jira project called RENOVATE. It sounds like we would need to wait for https://github.com/renovatebot/renovate/issues/3796.

Is there someone who can give a quick lay of the land on Jira (or Atlassian) support?

Pinging @setchy since it seems like they've done some deep digging into this. 

### What you tried so far.

Have not run anything as we're waiting for an environment to be spun up internally so we're not testing it against our "Production" Jira and Bitbucket.

**Comment by viceice** [comment, marked as answer]
there is no Jira support yet. so you only have the PR on bitbucket server.

on bitbucket cloud renovate can partly use bitbucket issues, but they also lack a lot of features (check boxes).

there is also no checkbox support on bitbucket server/ datacenter

**Reply by Cjewett** [reply]
Thank you for such a quick response! I've been taking a look at setchy's branch for Jira Cloud support I'll keep an eye on that one since it seems like the prerequisite to any Data Center or Server work being completed.

**Reply by setchy** [reply]
Hi @Cjewett - thanks for the tag.  https://github.com/renovatebot/renovate/pull/20559 certainly needs a little love - it was "functional" at the time, but required some refactoring to make it more reusable.  

I paused that PR for a moment while I focused on other feature enhancements, but should be able to pick this back up in the next month or so...
